python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -172,124 +178,46 @@ def get_in_progress_runs_by_step(graphene_info, job_names, step_keys):
     ]
 
 
-def get_asset_run_stats_by_step(graphene_info, job_names, asset_nodes):
-    # This is a utility method that gets the latest run that selected an asset,
-    # by searching within the last 5 runs of each job the asset belongs in.
-    # If none of the most recent runs selected the asset, we return back a GrapheneJobRunsCount
-    # object that contains the total number of job runs that have occurred since the latest
-    # asset materialization (or the total number of job runs if the asset has never been
-    # materialized).
+def get_asset_run_stats_by_step(graphene_info, asset_nodes):
```

Comment:
nit: maybe combine this with `get_latest_asset_run_by_step_key`?  doesn't seem like this is doing too much and it's the only caller...